runtime/src/mem/jemalloc/mem-jemalloc.c

```diff
@@ -43,9 +43,11 @@
 #endif
 
 static struct shared_heap {
+  chpl_bool isFixed;
```

Comment:
More descriptive name? Maybe `isFixedSize` or `isExtendable`? Oh, and maybe underscore instead of camelCase to match the others?

Reply:
Fixed page_size and isFixed in 51a7c566.